code/datums/diseases/_disease.dm

```diff
@@ -93,13 +95,16 @@
 	if(!. || (needs_all_cures && . < cures.len))
 		return 0
 
-/datum/disease/proc/spread(atom/source, force_spread = 0)
+
+/datum/disease/proc/spread(force_spread = 0)
```

Comment:
This isn't boolean. See a few lines below; it controls how far it will spread if `force_spread` is passed in an argument.